status-client-spec.md

```diff
@@ -109,10 +125,43 @@ v5](https://github.com/ethereum/devp2p/blob/master/discv5/discv5.md) and
 [modifications](https://github.com/status-im/rendezvous#differences-with-original-rendezvous)).
 Additionally, some static nodes MAY also be used.
 
-This part of the system is currently underspecified and requires further detail.
+A Status client MUST use at least one discovery method or use static nodes
+to communicate with other clients.
+
+Discovery V5 uses bootstrap nodes to discover other peers. Bootstrap nodes MUST support
```

Comment:
Can we generally clarify what the requirements are for each node type? I.e. as a bootstrap node, or as a mobile node etc.

Reply:
Yes but what kind of requirements would you like to distinguish? Protocols support, number of peers it seek to connect to or something more?

Reply:
Any requirement that someone writing a client, or mailserver server, might care about. Things they MUST do SHOULD do or MAY do.

E.g. MUST support x version or above and SHOULD be connected to N nodes and MAY choose to do action x